initScripts/aarch32/Ubuntu_16.04/Docker_18.03.sh

```diff
@@ -229,6 +230,14 @@ install_ntp() {
   fi
 }
 
+setup_restrictions() {
```

Comment:
We should name this `setup_docker_in_docker` so we don't end up overloading this function with more stuff later.